components/SideMenu/SideMenu.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <nav class="sidebar">
+    <template v-for="(item,i) in items">
+      <slot :name="item.key" :item="item">
+        <a :href="item.href" class="link-secondary" :key="i">
+          <i class="chevron right"></i>
+          {{ item.text }}
+        </a>
+      </slot>
+    </template>
+  </nav>
+</template>
+
+<script>
+export default {
+  name: "Sider",
```

Comment:
This should have the same name than the component
```suggestion
  name: "SideMenu",
```

Reply:
Done.